src/graphql/schema/publication.js

```diff
@@ -0,0 +1,9 @@
+export default `
+  type Publication {
+    id: ID!
+    name: String!
+    image: String!
+    enabled: Boolean
```

Comment:
This field is better be removed for now

Reply:
If we remove the field above, we won't be able to perform queries like this one
```graphql
{
  post(id: 1) {
    title
    publication {
      name
    }
  }
}
```

Reply:
I mean specifically the `enabled` property not the publication field